.github/workflows/build-and-push-image.yaml

```diff
@@ -0,0 +1,51 @@
+name: Build and push container images
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - "master"
+  push:
+    branches:
+      - "master"
```

Comment:
seem to require a mechanism to enforce branch name to apply ci-test properly.

Reply:
@Migorithm (cc. @JWSong @echo724 )

I notified this branch rules on README.md and PR description. thanks!